scripts/provision.sh

```diff
@@ -200,7 +201,22 @@ start_datastore() {
 }
 
 start_minio() {
+  set +e
+  is_minio_migration_needed
+  migration_needed=$?
+  set -e
```

Comment:
do we need these sets? Wouldnt `if is_minio_migration_needed; then` be enough?

Reply:
yeah we need these sets otherwise non zero exit codes will not be captured and the script will stop execution when return 1 is encountered